build-container.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
+
+function usage() {
+    echo "$0 <version>"
+}
+
+if [ -z "$1" ]; then
+    usage
+    exit 1
+fi
+
+VERSION="${1}"
```

Comment:
fyi: `VERSION=$1` is good enough; simple assignment isn't subject to globbing or word-splitting.